src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/TestRepository.cs

```diff
@@ -38,11 +38,6 @@ public void AddTest(Test test)
             this._dbContext.SaveChanges();
         }
 
-        public Test GetCurrentTest()
```

Comment:
why have you removed that method? we don't use it anymore?

Reply:
We used that logic only in unit tests, and it was using client site evaluation, so was invalid. Integration tests have detected this.